test/module/irohad/multi_sig_transactions/mst_test_helpers.hpp

```diff
@@ -103,13 +104,12 @@ inline auto makeTx(const shared_model::interface::types::CounterType &counter,
           .finish());
 }
 
-inline auto makePeer(const std::string &address, const std::string &pub_key) {
-  return std::make_shared<shared_model::proto::Peer>(
-      shared_model::proto::PeerBuilder()
-          .address(address)
-          .pubkey(shared_model::crypto::PublicKey(
-              shared_model::crypto::Hash::fromHexString(pub_key)))
-          .build());
+inline auto makePeer(const std::string &address,
```

Comment:
Maybe reuse this function where you create peer for in other files? It can be moved to a separate header.